en/docs/deploy/security/manage-keystores-with-the-ui.md

```diff
@@ -17,11 +17,11 @@ WSO2 Identity Server Management Console enables viewing keystore details such as
 
 ## Upload keystores  
 
-Follow the instructions below to upload a keystore file to via the Management Console.
+Follow the instructions below to upload a keystore file via the WSO2 Identity Server Management Console.
 
 <ol>
     <li>
-        <p>In the <b>Main</b> menu of the <a href="../../../deploy/get-started/get-started-with-the-management-console/">Management Console</a>, click <b>Manage > Keystores > Add</b>.</p>
+        <p>In the <b>Main</b> menu of the WSO2 Identity Server Management Console (<code>https://&lt;IS_HOST&gt;:&lt;PORT&gt;/carbon</code>), click <b>Manage > Keystores > Add</b>.</p>
```

Comment:
in some docs, we have not bolded `>`. Better if we can decide on a common format for this

Reply:
We can do a bulk replace and send a separate PR for this. Personally, I don't think it is a major inconsistency.

Reply:
IMO this is not something which we can achieve easily with a bulk replace

Reply:
As discussed, can do it with a few patterns. Will keep a note.